core/src/main/java/feast/core/util/ProtoUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.core.util;
+
+import com.google.protobuf.util.JsonFormat;
+import feast.core.FeatureSetProto;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProtoUtil {
+
+  public static String toJson(List<FeatureSetProto.FeatureSet> featureSets) throws IOException {
```

Comment:
Can we avoid creating non-generic utility methods. `ProtoUtil` and `toJson` seem like a generic class and method, but the implementation is specific to FeatureSetProtos. 

Either we need to rename this to be more specific and generalize later, or move this functionality out.

Reply:
Rename seems more of a band aid solution, so i refactored my commits such that it is no longer under util, and can be extended for other compression strategies.